uropa/visualization.py

```diff
@@ -108,30 +123,71 @@ def peak_count_plot(table, var, kind):
     pass
 
 
-def upset_plot(table, var):
+def upset_plot(table, var="feature", peak_columns=["peak_chr", "peak_start", "peak_end", "peak_strand"], title=None, title_size=20, path=None, dpi=300.0):
     """
     Visualize overlaps in a set with an upset plot.
     Similar to a Venn diagram but more readable especially with higher category count.
-
     For more information see: https://ieeexplore.ieee.org/document/6876017
-
     https://upsetplot.readthedocs.io/en/stable/
-
     Parameters
     ----------
     table : pd.DataFrame
         Pandas dataframe containing the data.
-    var : <datatype>, <default value>
-        <param description>
-    TODO add more parameters
-
+    var : String, default="feature"
+        Column which's values to list per peak and plot as categories.
+    peak_columns: List of Strings, default=["peak_chr", "peak_start", "peak_end", "peak_strand"]
+        Colums in table which identify peaks, to be used in group_by function.
+    title : String, default=None
+        Value for title of plot. If None plot has no title.
+    title_size: Integer, default=20
+        Value for size in points of figure title.
+    path : String, default=None

```

Comment:
Error for filenames without path e.g. `test.png`

Reply:
Should be fixed for both upset_plot() and count_plot() functions.